swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
 class Vendor:
-    pass
\ No newline at end of file
+        def __init__(self, inventory = None):
+            if not inventory:
+                inventory = []
+            self.inventory = inventory
```

Comment:
Looks good! 

Another way you might see this written is with a ternary, as:

`my_var = "some value" if some_condition else "other value"`

In this situation, that would look like

`self.inventory = [] if inventory is None else inventory`